docs/meetings/team/2022_11_01.md

```diff
@@ -0,0 +1,80 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 9:36am
+
+**End Time** 10:45am
+
+**Location** Discord
+
+**Date** Nov 01 2022
+
+## 👋 Attendence
+
+1. Veronica
+2. Matt
+3. Edouard
+
+***Attendence Comments***
+
+Logan Parker could not make the meeting because is having a surgery today.
+
+## 🧸 Check-In
```

Comment:
Missing Dad Joke. Not the end of the world but my disappointment is immeasurable

Reply:
The hope this dad joke treats you well